src/libraries/System.Runtime.Serialization.Primitives/ref/System.Runtime.Serialization.Primitives.cs

```diff
@@ -3,6 +3,8 @@
 // ------------------------------------------------------------------------------
 // Changes to this file must follow the https://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
+using System.Collections.ObjectModel;
```

Comment:
Sources for ref file do not use `usings`. Everything uses fully qualified type name instead. (You can consider regenerating the file using `/p:GenerateReferenceAssemblySource` (https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md)